packages/alfa-rules/test/sia-r83/rule.spec.tsx

```diff
@@ -0,0 +1,118 @@
+import { jsx } from "@siteimprove/alfa-dom/jsx";
+import { test } from "@siteimprove/alfa-test";
+
+import { Document, Element, Text } from "@siteimprove/alfa-dom";
+import { Option } from "@siteimprove/alfa-option";
+
+import R83, { Outcomes } from "../../src/sia-r83/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+const { isText } = Text;
+
+test("evaluate() passes a text node that truncates overflow using ellipsis", async (t) => {
+  const document = Document.of((self) => [
+    Element.fromElement(
+      <div style="overflow: hidden; white-space: nowrap; text-overflow: ellipsis;">
+        Hello world
+      </div>,
+      Option.of(self)
+    ),
+  ]);
```

Comment:
Completely unrelated, but given how often we use that pattern in test cases, we probably should have a helper:
```javascript
function documentFromElement(element) {
  return Document.of(self => [ Element.fromElement(element, Option.of(self)) ]);
}
```

Reply:
I do agree that it has become a little repetitive. I haven't found a good abstraction yet though and actually prefer spelling it out over using a helper, simply because it does so little but requires yet another import 🤔